### ISSUE TITLE
navigation bar covering banner

### ISSUE THREAD

**Issue by PhranG** [ForeverFamilyFoundaton/website-v2#116]
Top navigation bar covers the banner on page: https://foreverfamilyfoundation.org/pages/afterlife-science

**Comment by dinshaw** [comment]
Right now, this will happen if your font-size has been bumped up.
I will work on a better layout next session.

**Comment by PhranG** [comment]
Can we readdress the banner issue? I know on my devices the banner cannot be seen, so I imagine that others have the same issue. Thanks

**Comment by dinshaw-he** [comment]
Yes, sorry for not making any updates for a while. I will find some time in the next couple weeks.
